LEAF_Nexus/templates/print_subindicators_ajax.tpl

```diff
@@ -33,13 +33,13 @@
                 </span>
         <!--{/if}-->
         <!--{if $indicator.format == 'text'}-->
-            <span class="printResponse" id="data_<!--{$indicator.indicatorID}-->_<!--{$categoryID}-->_<!--{$uid}-->">
-                <!--{$indicator.data}-->
+            <span class="printResponse" id="data_<!--{$indicator.indicatorID|strip_tags}-->_<!--{$categoryID|strip_tags|escape}-->_<!--{$uid|strip_tags|escape}-->">
+                <!--{$indicator.data|strip_tags}-->
```

Comment:
As we've very recently discovered, when dealing with `indicator` data, it's possible it contains valid HTML markup, so stripping the tags would remove that valid input. As of right now, we don't have a good way to determine if data should be stripped/escaped/unescaped. So, for now, the best thing to do is nothing when it comes to an `indicator.format` of text. We'll have to revisit this when we come up with a good solution.

This is only a consideration since the `indicator.format` is "text".